IdentityCore/src/oauth2/token/MSIDPrimaryRefreshToken.m

```diff
@@ -174,4 +180,23 @@ - (BOOL)isDevicelessPRT
     return prtVersion >= 3.0 && [NSString msidIsStringNilOrBlank:self.deviceID];
 }
 
+- (BOOL)shouldRefreshWithInterval:(NSUInteger)refreshInterval
```

Comment:
How will the fact that the interval is configurable be used in the implementation of the PRT refresh? For example, will a constant refresh interval be stored in a different place? Or will different flows make use of different refresh intervals?

Reply:
Yes, broker will provide a refresh interval. For now it will be constant, but could be configurable by server in the future. I wanted it to be controlled by the broker though.